### PR TITLE
Feature/new bindings

### PR DESCRIPTION
**Pull request by Manuthor**

### CODE REVIEW

File: scripts/download_wasm.py [hunk 1]
```diff
@@ -68,7 +72,8 @@ def download_wasm(version: str) -> bool:
         print(f'Cannot get cloudproof_rust {version} ({exception})')
         return False
 
+
 if __name__ == '__main__':
     RET = download_wasm('v2.1.0')
     if RET is False and getenv('GITHUB_ACTIONS'):
-        download_wasm('last_build/add_logs_on_findex_callbacks')
+        download_wasm('last_build/update_cover_crypt_12')
```
Comment (ccorsin) [inline scripts/download_wasm.py:79]: why is it named _12?

File: src/ecies/ecies.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import init from "../pkg/ecies/cloudproof_ecies"
```
Comment (ccorsin) [inline src/ecies/ecies.ts:1]: call this init.ts as in aes256gcm ? or the other way around, what's more homogeneous with the other functions :)

File: tests/ecies.test.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+/* Importing the functions from the ECIES library. */
+const { EciesSalsaSealBox } = await Ecies()
+
+test("Ecies SalsaSealBox", async () => {
+  const keyPair = EciesSalsaSealBox.generateKeyPair()
+
+  for (let plaintextSize = 1; plaintextSize <= 100; plaintextSize++) {
+    const plaintext = randomBytes(plaintextSize)
+    const authenticatedData = "authenticatedData"
```
Comment (ccorsin) [inline tests/ecies.test.ts:14]: why not looping here for authenticatedData as above ?

File: scripts/download_wasm.py (hunk 1 above)
Comment (Manuthor) [reply scripts/download_wasm.py:79]: it is the git branch name (for covercrypt 12.0). While it is not released yet, CI use this cloudproof_rust build to test JS code

**Review by ccorsin — APPROVED** [review]

**Pull request merged**